.github/workflows/docker-branch-releases.yml

```diff
@@ -10,13 +10,15 @@ on:
     paths-ignore:
       - "CHANGELOG.md"
       - "common/lib/dependabot/version.rb"
-  pull_request:
+  pull_request_review:
+    types: [submitted]
     paths-ignore:
       - "CHANGELOG.md"
       - "common/lib/dependabot/version.rb"
 
 jobs:
   push-updater-image:
+    if: github.event.review.state == 'approved'
```

Comment:
It doesn't look like the review is included in a push payload? https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

Reply:
`github.event_name == 'push'` might work here? 

https://docs.github.com/en/actions/learn-github-actions/contexts#example-contents-of-the-github-context

Reply:
True, I just broke the push again that I got working this morning. 😅